src/entities/task.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm'
+import { TaskList } from './taskList'
+
+@Entity()
+export class Task {
```

Comment:
I think there is a confusion between task and tasklist ?
Task must have a rank and a isDone attribute. TaskList do not need isDone.